homeassistant/components/osoenergy/services.yaml

```diff
@@ -237,6 +237,20 @@ set_v40_min:
           max: 550
           step: 1
           unit_of_measurement: L
+turn_away_mode_on:
+  target:
+    entity:
+      domain: water_heater
+  fields:
+    duration_days:
+      required: false
```

Comment:
I would argue that it should be required to avoid having 2 ways to do the same thing

Reply:
You mean make the duration_days required and leave the users to use the build in service to enable it without providing the duration?

Reply:
Yes!